### PR TITLE
Wrapping Radio Group in ChartSettings

### LINKED ISSUE #11241: Visualization settings sidebar doesn't account for longer translations
I'm trying out this idea of filing issues when I run into something in the FE codebase that feel like I should be able to fix, but can't because I run into something hard/odd.

I was attempting to refactor the left `ViewSidebar` to have an auto width in order to allow the tabs in `ChartSettingSidebar` to have enough width to fit them in cases where there are more than four tabs, or when using a language other than English that causes the tabs to be wide, like in this case:

image

The problem I ran into was in trying to do this in `View.jsx`:

```
<ViewSidebar left width={"auto"} isOpen={!!leftSideBar}>
  {leftSideBar}
</ViewSidebar>
```

This sort of worked, but it looked to me like `ViewSidebar` depends on their being an integer value for the width in order to compute the transition animation, and I was unsure how to pass "auto" as the width prop to this component without breaking the Motion.

```
const ViewSideBar = ({ left, right, width = 355, isOpen, children }) => (
  <Motion
    defaultStyle={{ opacity: 0, width: 0 }}
    style={
      isOpen
        ? { opacity: spring(1), width: spring(width, SPRING_CONFIG) }
        : { opacity: spring(0), width: spring(0, SPRING_CONFIG) }
    }
  >
```

### PR DESCRIPTION
**Pull request by npfitz**
Fixes #11241 

Small addition to the Radio component for whether or not the items should be wrapped (defaults to `false`). Wrapping radio buttons will add a bottom margin to all items in the `RadioGroup`, so the container below the chart setting sections was adjusted to compensate.

image

### CODE REVIEW

File: frontend/src/metabase/core/components/Radio/Radio.tsx [hunk 1]
```diff
@@ -55,6 +55,7 @@ export interface RadioProps<TValue extends Key, TOption = RadioOption<TValue>>
   showButtons?: boolean;
   onChange?: (value: TValue) => void;
   onOptionClick?: (value: TValue) => void;
+  wrap: boolean;
```
Comment (kulyk) [inline frontend/src/metabase/core/components/Radio/Radio.tsx:58]: Can we actually just pass the desired styles via emotion? See an example how [`DatasetFieldMetadataSidebar` styles `SelectButton's` nested component.](https://github.com/metabase/metabase/blob/91a87d814fa20125bacddf1d72ffd8e9dfe7a5f6/frontend/src/metabase/query_builder/components/DatasetEditor/DatasetFieldMetadataSidebar/DatasetFieldMetadataSidebar.styled.jsx#L33). You'll need to assign these internal visual components to `Radio`, see [how `SelectButton` does that](https://github.com/metabase/metabase/blob/91a87d814fa20125bacddf1d72ffd8e9dfe7a5f6/frontend/src/metabase/core/components/SelectButton/SelectButton.tsx#L85)

Comment (ranquild) [reply frontend/src/metabase/core/components/Radio/Radio.tsx:58]: Makes sense to me, given the component API.

**Comment by npfitz** [comment]
Updated the PR. I needed to change the export to make it work. I found the one other reference to `Radio.RadioGroup` and updated it as well.

File: frontend/src/metabase/core/components/Radio/Radio.tsx [hunk 2]
```diff
@@ -223,5 +222,11 @@ function isDefaultOption<TValue>(
...
 
 export default Object.assign(Radio, {
-  RadioGroup,
+  RadioGroupVarients: [RadioGroupBubble, RadioGroupNormal],
+  RadioLabelVarients: [RadioLabelBubble, RadioLabelNormal, RadioLabelText],
+  RadioContainerVarients: [
+    RadioContainerBubble,
+    RadioContainerNormal,
+    RadioContainerUnderlined,
+  ],
```
Comment (kulyk) [inline frontend/src/metabase/core/components/Radio/Radio.tsx:231]: Should it be "variants"?

Comment (npfitz) [reply frontend/src/metabase/core/components/Radio/Radio.tsx:231]: Yes... Spelling is hard

Comment (npfitz) [reply frontend/src/metabase/core/components/Radio/Radio.tsx]: Fixed

**Review by kulyk — APPROVED** [review]
LGTM! Would appreciate a look from @ranquild on that too

**Review by ranquild — APPROVED** [review]
I would prefer `Radio.Group` rather than `Radio.RadioGroup`, but it's just a personal preference.

**Pull request merged**